07-documenting_R.Rmd

```diff
@@ -70,17 +70,15 @@ The language used in the documentations should follow these basic rules:
 
 4. Giving a code equivalent (or approximate equivalent) can be a useful in addition to the description provided. You should carefully weigh whether the code equivalent adds value to the document.
 
-5. The tone of the documentation needs to be respectful of the reader’s background. Lay out the relevant information, show motivating use cases, provide glossary links, and do your best to connect-the-dots. The documentation is meant for newcomers, many of whom will be using it to evaluate the R language as a whole. The experience needs to be positive and not leave the reader with worries that something bad will happen if they make a mistake. 
-
-Extensive details of writing R documentation files can be found [here](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files).
+5. The tone of the documentation needs to be respectful of the reader’s background. Lay out the relevant information, show motivating use cases, and do your best to connect-the-dots. The documentation is meant for newcomers, many of whom will be using it to evaluate the R language as a whole. The experience needs to be positive and not leave the reader with worries that something bad will happen if they make a mistake. 
 
 ## Helping with documentation
 
 Maintaining the accuracy  of R's documentations and keeping a high level of quality takes a lot of effort. Community members, like you, help with writing, editing, and updating content, and these contributions are appreciated and welcomed.
 
 Looking at pre-existing documentation source files can be very helpful when getting started.
 
-If you look at documentation issues on Bugzilla, you will find various documentation problems that may need work. Issues vary from typos to unclear documentation and items lacking documentation.
+If you look at documentation issues on Bugzilla, you will find various documentation problems that may need work. Since "Documentation" is one of the possible components you can set in the metadata, you can browse by this [component](https://bugs.r-project.org/bugzilla/buglist.cgi?component=Documentation) to find the documentation bugs on Bugzilla. Issues vary from typos to unclear documentation and items lacking documentation.
```

Comment:
I feel this has gotten overly complicated. The relevant info is: you can directly search for docs issues on Bugzilla + maybe a link to such a view. Using Bugzilla teminology such as "component" and explaining that this is a piece of metadata attached to each bug report is besides the point. Just my 2 ¢...